docs/source/releases/latest/new-prd-fam-unsectored-xarray-dict-to-algorithm-to-output-format.yaml

```diff
@@ -0,0 +1,24 @@
+performance:
+- description: |
+    This branch supports a new product/product_defaults family called
```

Comment:
This is a nitpick. It's not that "this branch" supports new functionality. You are submitting a change that supports new functionality. So, this should be written from that perspective. For example, "Add support for a new ...".

Sorry for the nitpick, but it will help make release notes more readable.

Reply:
I'll update this!

Reply:
Done